runtime/balrt.h

```diff
@@ -8,6 +9,11 @@
 #define UT_MASK 0x1F
 #define TAG_SHIFT 56
 
+#define UN -1
```

Comment:
These are a bit short for something with global scope. I suggest adding a `COMPARE_` prefix.

Reply:
Fixed